Source/ProgressionSystemRuntime/Private/Components/PSSpotComponent.cpp

```diff
@@ -81,3 +134,19 @@ void UPSSpotComponent::ChangeSpotVisibilityStatus(UMySkeletalMeshComponent* Mesh
 	FPSSaveToDiskData SaveToDiskDataRow = UPSWorldSubsystem::Get().GetCurrentSaveToDiskRowByName();
 	Mesh->SetActive(!SaveToDiskDataRow.IsLevelLocked);
 }
+
+// Refresh Amount Of Unlocked skins for the character (level)s
+void UPSSpotComponent::RefreshAmountOfUnlockedSkins(bool bApplySkin)
+{
+	UMySkeletalMeshComponent& SpotMeshComponent = GetMeshChecked();
+	int32 UnlockedSkinsAmount = UPSWorldSubsystem::Get().GetCurrentSaveToDiskRowByName().UnlockedSkinsAmount;
+
+	for (int Index = 0; Index < UnlockedSkinsAmount; Index++)
```

Comment:
- `int` is not allowed in Unreal Engine coding standards; use `int32` instead.
- Using `int` can lead to platform-dependent behavior and potential compatibility issues.

Please always use `int32`-only

Reply:
my bad:  https://github.com/moinm3uw/ProgressionSystem/pull/59/commits/911f689c0d341e28064f0e7da53db4cb67e6afc7